src/component/typingbox/__tests__/TypingBox.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import TypingBox from '../TypingBox';
+import renderer from 'react-test-renderer';
+import { mount, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
+
+describe("TypingBox", () => {
+  it("renders a textbox", () => {
+    const TypingBoxComponent = renderer.create(<TypingBox/>).toJSON();
```

Comment:
I would probably include all of the props that this component needs for each test. We don't really care what renders when we send it no props, as this will likely never happen in our application. Using some sort of `buildProps` function can help clean this up a lot.

We haven't talked about it yet, but there's something we can use called PropTypes which lets us define whether or not a prop is required.